bindata/network/additional-networks/openshift-sriov/sriov-cni.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Does it make sense to put this as another container in the multus daemonset?

Reply:
That would be my preference.

Reply:
(also, not really related to this PR, but would it make sense to copy the multus initcontainers into the openshift-sdn daemonset, so that if you're running openshift-sdn, we don't actually need to deploy the multus daemonset too?)

Reply:
If Multus is not enabled, do we support using openshift-sdn together with CNI plugins from those initcontainers?  If not,  then there might be less willingness to copy those CNI plugins in openshift-sdn.

Reply:
I agree that this could be achieved easily by passing a env var to enable/disable multus deployment. Considering later when adding ovn-kubernetes support, it may require adding multus initcontainers there as well because ovn could be one of the default network plugins.